client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,60 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: white;
+  border-width: 0px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  border: 2px solid black;
+  border-radius: 8px;
+  border-collapse: separate;
+  border-spacing: 0;
+  display: block;
+  overflow-x: auto;
+  overflow-y: auto;
+  height: 100%;
+  width: 100%;
+}
+
+.scoreboard tbody td {
+  border-bottom: 1px solid #ddd;
+  border-right: 1px solid #ddd;
+  color: #333;
+  padding: 5px;
+  text-align: right;
+  white-space: nowrap;
+  background-color: #f8f8f8;
+}
+
+.scoreboard tbody tr:nth-child(1) td:nth-child(1) {
+  z-index: 2;
+}
+
+.scoreboard tbody tr:nth-child(1) td {
+  position: sticky;
+  z-index: 1;
+  top: 0;
+  background-color: #ddd;
+}
```

Comment:
sorry I have no knowledge off tables CSS so I don't even know if I'm marking the correct spot.

I would really like if the columns could cover all the widget. I wouldn't mind if the player names would be cut but I really dont like how it goes from this: 
![image](https://user-images.githubusercontent.com/73476656/207125766-a1bd7261-d075-45e9-b794-3b3b068d0c5b.png)

to this:

![image](https://user-images.githubusercontent.com/73476656/207126220-ce4f74ca-9058-4b4e-9314-fa3cd101258d.png)

Reply:
I'm not asking to resize the whole widget. I'm asking to make the size of the columns fit the whole widget in the size I set up. something similar to what would happen if you resized a table in word

Reply:
I imagine there is a way to do that.  You know the size of the widget.  You can count how many columns there are are (first column plus _n_ seats.  Do the math and set the width of each column to the result.  But as I demonstrated in earlier tests, with column width flexibility, if someone has a long name it will spread out and create a scrollbar (which you said you don't care about, so maybe that is a solution).

Alternatively, use some of the css tricks I show in the Tutorial and get rid of the border around the scoreboard and make the background transparent.  Then you don't see the "extra."  The problem with that is if a scrollbar is needed, it looks strange.

Reply:
> This has been cooking for quite a while now

Don't worry I'm not requesting any sort of redo.

I just wanted to add a feature to be able to do what I describe in this other comment.
https://github.com/ArnoldSmith86/virtualtabletop/pull/1381#discussion_r1046638529

maybe the CSS in that scoreboard could be a class 🤷

>  what should happen if there are ten players in a 100 pixel-wide scoreboard

this would be defined by the game designer so players should not encounter this kind of situation

> Finally, what game designer controls would you want around all of this?

As I said in the other comment only a property or CSS variable would be great as is allows CSS calculation for the sizes of the collumns